partiql-spi/src/main/kotlin/org/partiql/spi/function/builtins/FnLte.kt

```diff
@@ -88,8 +88,8 @@ internal val Fn_LTE__DECIMAL_ARBITRARY_DECIMAL_ARBITRARY__BOOL = Function.static
     name = "lte",
     returns = PType.bool(),
     parameters = arrayOf(
-        @Suppress("DEPRECATION") Parameter("lhs", PType.decimal()),
-        @Suppress("DEPRECATION") Parameter("rhs", PType.decimal()),
+        Parameter("lhs", PType.decimal()),
```

Comment:
Similarly, fn name variable uses DECIMAL_ARB. Perhaps should get changed?

Also applies for
- lt
- gt
- gte
- between
- may have some others, so probably need to do a check on Builtins.kt